handbook/engineering/security/reporting-vulnerabilities.md

```diff
@@ -0,0 +1,59 @@
+# Reporting a Vulnerability
+
+If you think that you have found a security or privacy vulnerability, please email us at <a href="mailto:security@sourcegraph.com">security@sourcegraph.com</a>. We will reply to reports within 2 US business days to acknowledge that we received them, and will strive to send you regular updates on our progress until the issue is resolved. You may request an update by replying to the existing email thread. We will read, but may not respond to low quality or spammy reports (e.g. those produced by automated tooling). We welcome reports from everyone, including security researchers, developers, and customers. 
+
+### Bounties
+
+We provide monetary rewards, from \$50 to \$10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, and the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit.
+
+We will send payment to a valid PayPal account. We will ask you for the name and country associated with your PayPal account.
+
+**Timelines**
```

Comment:
In the first section you say we respond within 2 US business days, but it is unclear in this table whether these are also all US business days.